foundation/donate.dd

```diff
@@ -30,6 +30,18 @@ $(H3 How can I donate?)
 
 We $(LINK2 sponsors.html, welcome contributions) from both organizations and individuals. Your help is highly appreciated.
 
+$(DONATE_ITEM Donate through OpenCollective, users,
+
+    Make a $(I transparent) donation. Select the package you're comfortable with and get the T-shirt!
```

Comment:
This makes is sound like any donation gets a tshirt. I'd just leave this sentence out.

Reply:
Yes!

Reply:
Fixed.